### PR TITLE
[FIX] STOCK: incorrect style in informations section in deliveryslip report

### PR DESCRIPTION
**Pull request by RodrigoBM**
Description of the issue/feature this PR addresses:

This PR fixes the incorrect styling applied by the css when reporting the client_order_ref field.

Current behavior before PR:

Selección_990

Desired behavior after PR is merged:

Selección_989

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You should rebase over odoo/odoo remote instead of OCB

**Comment by RodrigoBM** [comment]
> You should rebase over odoo/odoo remote instead of OCB

Done, i dont check remote

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hello, thanks for the fix :)

Could you please keep the `t-if` and `class` attributes in their previous order? Keeping `t-if`s first really helps with understanding when scanning the code.

**Comment by RodrigoBM** [comment]
> Hello, thanks for the fix :)
> 
> Could you please keep the `t-if` and `class` attributes in their previous order? Keeping `t-if`s first really helps with understanding when scanning the code.

Done

**Comment by AntoineVDV** [comment]
@robodoo delegate=vchu-odoo

File: addons/delivery/views/report_deliveryslip.xml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 <odoo>
     <template id="report_delivery_document2" inherit_id="stock.report_delivery_document">
         <xpath expr="//div[@name='div_sched_date']" position="after">
-            <div t-if="o.picking_type_id.code == 'outgoing' and o.carrier_id" class="col-auto">
+            <div t-if="o.picking_type_id.code == 'outgoing' and o.carrier_id" class="col-auto col-3 mw-100 mb-2">
                 <strong>Carrier:</strong>
-                <p t-field="o.carrier_id"/>
+                <p class="m-0" t-field="o.carrier_id"/>
```
Comment (vchu-odoo) [inline addons/delivery/views/report_deliveryslip.xml:6]: Could you please put `t-field` before `class` like this everywhere?
```suggestion
                <p t-field="o.carrier_id" class="m-0"/>
```

File: addons/stock/report/report_deliveryslip.xml [hunk 2]
```diff
@@ -45,19 +45,19 @@
                     <h2>
                         <span t-field="o.name"/>
                     </h2>
-                    <div class="row mt32 mb32">
-                        <div t-if="o.origin" class="col-auto" name="div_origin">
+                    <div id="informations" class="row mt-4 mb-4">
```
Comment (vchu-odoo) [inline addons/stock/report/report_deliveryslip.xml:48]: why do you need `id` here?

**Review by vchu-odoo — CHANGES_REQUESTED** [review]
Hey, thank you for the fix!

Here are some small comments.
Is it possible to reduce the margin between fields here?
image

File: addons/stock/report/report_deliveryslip.xml (hunk 2 above)
Comment (RodrigoBM) [reply addons/stock/report/report_deliveryslip.xml:48]: I added it, to homogenize it with other reports. 

- https://github.com/odoo/odoo/blob/16.0/addons/sale/report/ir_actions_report_templates.xml#L43
- https://github.com/odoo/odoo/blob/16.0/addons/account/views/report_invoice.xml#L63

File: addons/delivery/views/report_deliveryslip.xml (hunk 1 above)
Comment (RodrigoBM) [reply addons/delivery/views/report_deliveryslip.xml:6]: OK, I can change this, but it does not follow the standard of other reports.

- https://github.com/odoo/odoo/blob/16.0/addons/sale/report/ir_actions_report_templates.xml#L46
- https://github.com/odoo/odoo/blob/16.0/addons/account/views/report_invoice.xml#L69

Comment (RodrigoBM) [reply addons/delivery/views/report_deliveryslip.xml:6]: Do you want me to change it?

**Comment by RodrigoBM** [comment]
> Hey, thank you for the fix!
> 
> Here are some small comments. Is it possible to reduce the margin between fields here? image

I think it is not possible as we apply col-3 to all segments. If you have any idea how to do it, I'll try it.

File: addons/delivery/views/report_deliveryslip.xml (hunk 1 above)
Comment (RodrigoBM) [reply addons/delivery/views/report_deliveryslip.xml]: Done

**Review by vchu-odoo — APPROVED** [review]
Thank you for the changes!

LGTM :+1:

**Comment by vchu-odoo** [comment]
@robodoo r+

**Comment by vchu-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**